sql/create-db.sql

```diff
@@ -0,0 +1,76 @@
+-- Set up a PostgreSQL Database for use as the backing
```

Comment:
Ok, sounds good. Maybe we should find an expert and quiz them sometime in the next couple weeks.